variants/meshlink_eink/platformio.ini

```diff
@@ -0,0 +1,128 @@
+; The very slick RAK wireless RAK 4631 / 4630 board - Unified firmware for 5005/19003, with or without OLED RAK 1921
```

Comment:
The comments from the non-eink variant are valid here too. I won't repeat them ;-)